webxdc.d.ts

```diff
@@ -77,6 +77,13 @@ interface Webxdc<T> {
     cb: (statusUpdate: ReceivedStatusUpdate<T>) => void,
     serial?: number
   ): Promise<void>;
+
+  /**
+   * Set a listener for _ephemeral_ status updates.
+   * Own status updates are not received.
```

Comment:
Does this trigger a regular message to announce the peer-address? 
Does this wait on anything? 
IOW: the docs are not clarifying what a caller can expect in terms of how connectivity happens (same question for `setEphemeralUpdateListener`.

Reply:
Desktop explicitely calls [send_gossip_advertisement](https://github.com/deltachat/deltachat-core-rust/blob/becc3ebd4f54a4c310126e8e9544c6bc364d70cd/src/peer_channels.rs#L214) to send out an advertisiment with the current IP address. The async call resolves as soon as at least one peer is connected and the swarm is operatable.